cpp/test/sg/fil_test.cu

```diff
@@ -15,9 +15,12 @@
  */
 
 #include <cuml/fil/fil.h>
+#include <cuml/fil/multi_reduction.h>
```

Comment:
As `multi_reduction` is defined in a separate file, consider moving the test into a separate file as well.